fdbserver/MetricLogger.actor.cpp

```diff
@@ -364,41 +375,275 @@ ACTOR Future<Void> updateMetricRegistration(Database cx, MetricsConfig* config,
 	}
 }
 
-ACTOR Future<Void> runMetrics(Future<Database> fcx, Key prefix) {
-	// Never log to an empty prefix, it's pretty much always a bad idea.
-	if (prefix.size() == 0) {
-		TraceEvent(SevWarnAlways, "TDMetricsRefusingEmptyPrefix").log();
-		return Void();
+// ACTOR Future<Void> runMetrics(Future<Database> fcx, Key prefix) {
```

Comment:
I think it should be fine to leave this existing `runMetrics` function in, since it has different parameters than the one you added and isn't called anywhere.

Reply:
It was initially called in `workerServer` but I did change that part to the new `runMetrics`

Reply:
I'll add them back